### PR TITLE
[jellyfin] Move image to official

### PR DESCRIPTION
**Pull request by CuBiC3D**
**Description of the change**

Moving image from ls.io to official docker image and added values docs for rootless GPU transcoding from Plex.

**Benefits**

- no s6 overlay
- semver

**Possible drawbacks**

Folder structure of `/config` and `/cache` will change and break config.
As discussed, we keep the official structure and provide rough migration instructions if user does not want to start over.

**Applicable issues**

none

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [x] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/stable/jellyfin/values.yaml [hunk 1]
```diff
@@ -41,3 +59,28 @@ persistence:
...
+#   # $ cat /etc/group | grep "video\|render"
+#   # video:x:44:
+#   # render:x:109:
+#   supplementalGroups:
+#   - 44
+#   - 109
+
+# resources:
+#   requests:
+#     # Hardware acceleration using an Intel iGPU w/ QuickSync
```
Comment (onedr0p) [inline charts/stable/jellyfin/values.yaml:79]:
```suggestion
#     # Hardware acceleration using an Intel iGPU w/ QuickSync and
#     # using intel-gpu-plugin (https://github.com/intel/intel-device-plugins-for-kubernetes)
```

File: charts/stable/jellyfin/values.yaml [hunk 2]
```diff
@@ -41,3 +59,28 @@ persistence:
...
+#     gpu.intel.com/i915: 1
+#     cpu: 200m
+#     memory: 256Mi
+#   limits:
+#     # Hardware acceleration using an Intel iGPU w/ QuickSync
```
Comment (onedr0p) [inline charts/stable/jellyfin/values.yaml:84]:
```suggestion
#     # Hardware acceleration using an Intel iGPU w/ QuickSync and
#     # using intel-gpu-plugin (https://github.com/intel/intel-device-plugins-for-kubernetes)
```

**Review by onedr0p — CHANGES_REQUESTED** [review]
I need to update this on the plex-media-server-chart as well!

**Comment by CuBiC3D** [comment]
Waiting for the `hack/gen-helm-docs.sh` fix to support new repo structure (stable/incubator)

**Review by onedr0p — APPROVED** [review]

**Pull request merged**